File: lec01q2.cpp
#include<iostream>
using namespace std;
int main(){
    cout<<"enter amount:"<<endl;
    int amount=0;
    cin>>amount;
    cout<<"enter rate:"<<endl;
    int rate=0;
    cin>>rate;
    cout<<"enter time:"<<endl;
    int time=0;
    cin>>time;
    int si=(amount*rate*time)/100;
    cout<<"simple intrest is:"<<si<<endl;
    return 0;
}

File: lec01q2.java
import java.util.Scanner;
class lec01q2{
    public static void main(String[] args){
        
        Scanner scn = new Scanner(System.in);

        System.out.println("enter amount:");
        int amount = scn.nextInt();
        System.out.println("enter rate:");
        int rate = scn.nextInt();
        System.out.println("enter time:");
        int time = scn.nextInt();
        int si= (amount * rate * time)/100;
        System.out.println("simple interst is:" + si);
    }


}